build-system/tasks/extension-helpers.js

```diff
@@ -666,11 +678,15 @@ async function buildNpmBinaries(extDir, name, options) {
     };
     if (options.useBentoCore) {
       // remap all shared modules to the @bentoproject/core package and declare as external
-      npm.bento.remap = getRemapBentoNpmDependencies();
+      // TODO(wg-bento): Add `external` values as dependencies on package.json
```

Comment:
@kvchari Where do we add `@bentoproject/core` at the moment?

Reply:
@bentoproject/core is added as a dependency of each extension in the write-package-files.json